/*
 * This class' responsibility is to denote when an Entry couldn't be found.
 * It's thrown by KeyStoreWrapper.
 */
package passwordvault.security.vault;

/**
 * This class is thrown when an entry wasn't found in the KeyStore.
 */
public class InstanceNotFoundException extends Exception {
    
}
